.gitignore

```
@@ -127,3 +127,9 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# simualtor for signals - not for git
+test_signal_gen.ipynb
+
+# docker compose
+docker-compose.yaml
```

Remember to add linebreak at the end of the file
```suggestion
docker-compose.yaml

```